try_bro/plugin.spec.yaml

```diff
@@ -3,21 +3,23 @@ extension: plugin
 product: ["insightconnect"]
 name: try_bro
 title: Try Bro
-description: Try the Bro Network Security Platform
+description: "With the Try Bro plugin for Rapid7 InsightConnect, users can use a free instance of Bro Network Security
+  Monitor in the cloud. Users can upload PCAP files for analysis and get Bro logs"
 version: 1.0.1
 vendor: rapid7
 support: community
 status: []
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/try_bro
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.bro.org
 tags:
 - bro
 - nsm
 - pcap
 - ids
 hub_tags:
-  use_cases: []
+  use_cases: [log_search, alert_enrichment, application_security, data_ingestion, detection_and_response, log_search, network_containment]
```

Comment:
detection_and_response doesn't exist, it should be threat_detection_and_response
Remove network containment, data ingestion, alert_enrichment, and application_security
Add threat hunting, investigations